queries.sql

```diff
@@ -1,3 +1,51 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+
+SELECT *
+FROM animals
+WHERE name LIKE '%mon';
+
+
+-- List the name of all animals born between 2016 and 2019.
+
+SELECT *
+FROM animals
+WHERE date_of_birth > '2016-01-01' and date_of_birth < '2019-12-31';
```

Comment:
- [ ] In this query you should only select names of animals with data of birth between 2016 and 2019, kindly make sure to return only the names, not all the table data

Reply:
Done!